src/style.css

```diff
@@ -1,3 +1,93 @@
 body {
   margin: 0;
+  background-color: grey;
+  font-family: Roboto,sans-serif;
+
+  --header-height: 60px;
+  --active-color: #00ACDC;
+}
+
+.header {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  flex-shrink: 0;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+}
+
+.moyo__logo {
+  display: flex;
+  position: relative;
+  left: 50px;
+}
+
+.nav {
+  display: flex;
+  position: relative;
+  right: 50px;
+
+}
+
+.nav__list {
+  display: flex;
+  flex-shrink: 0;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  position: relative;
+}
+
+
+
+.nav__link {
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: var(--header-height);
```

Comment:
set a correct line height from figma and to get height you need you can just use padding